CMakeLists.txt

```diff
@@ -25,18 +24,31 @@ set(SOURCES
 
 add_library(xdf ${SOURCES})
 
+find_package(pugixml QUIET)
+if(TARGET pugixml AND NOT XDF_NO_SYSTEM_PUGIXML)
+	message(STATUS "Using system pugixml")
+	target_link_libraries(xdf PRIVATE pugixml)
+else()
+	message(STATUS "Using bundled pugixml")
+	target_sources(xdf PRIVATE pugixml/pugixml.cpp)
+	target_include_directories(xdf PRIVATE pugixml)
+endif()
+
```

Comment:
This looks a bit risky; not sure how much difference between different pugixml versions, but things might break.

Reply:
Differences in minor versions should be unproblematic and this is mostly aimed at linux distributions that will patch in something like this anyways. I'll add a minimum version requirement, just to be on the safe side.

Reply:
Sounds good.

Reply:
Added in d7eb241